common/elasticsearch/mapping.go

```diff
@@ -286,6 +290,10 @@ var mapping = `{
           "type": "string",
           "index": "not_analyzed"
         },
+        "Cluster": {
```

Comment:
I think we should call it `ClusterName` and not just cluster.. it'll be easier to understand imo